source/NVDAObjects/IAccessible/MSHTML.py

```diff
@@ -954,15 +954,15 @@ def setFocus(self):
 
 	def _get_table(self):
 		if self.role not in (controlTypes.ROLE_TABLECELL,controlTypes.ROLE_TABLEROW) or not self.HTMLNode:
-			raise NotImplementedError
+			return None
 		HTMLNode=self.HTMLNode
 		while HTMLNode:
 			nodeName=HTMLNode.nodeName
 			if nodeName:
 				nodeName=nodeName.upper()
 			if nodeName=="TABLE": return MSHTML(HTMLNode=HTMLNode)

```

Comment:
This does not support aria based tables (e.g. role grid). I don't think that's a problem, though it might deserve a change or a comment.

Reply:
I am not much aware of the aria based tables support, but please feel free to propose any useful comment to insert here.
As long as this PR does not change a thing in that scope - that is, no client code is expecting a `NotImplementedError` - I would suppose that further refining the process is out of the scope of this PR, don't you think?